src/main/java/de/tum/cit/aet/artemis/athena/service/AthenaFeedbackSuggestionsService.java

```diff
@@ -102,65 +102,77 @@ private record ResponseDTOModeling(List<ModelingFeedbackDTO> data, ResponseMetaD
     /**
      * Calls the remote Athena service to get feedback suggestions for a given submission.
      *
-     * @param exercise   the {@link TextExercise} the suggestions are fetched for
-     * @param submission the {@link TextSubmission} the suggestions are fetched for
-     * @param isGraded   the {@link Boolean} should Athena generate grade suggestions or not
+     * @param exercise      the {@link TextExercise} the suggestions are fetched for
+     * @param submission    the {@link TextSubmission} the suggestions are fetched for
+     * @param isPreliminary the {@link Boolean} should Athena generate grade suggestions or not
```

Comment:
Do you think this renaming make sense? What is the reason behind - because then on API level the parameters are still called isGraded and are negated before being passed to the function call